lib/fast/cli.rb

```diff
@@ -14,14 +14,16 @@ module Fast
 
   # Highligh some source code based on the node.
   # Useful for printing code with syntax highlight.
-  def highlight(node, show_sexp: false)
+  def highlight(node, show_sexp: false, colorize: true)
     output =
       if node.respond_to?(:loc) && !show_sexp
         node.loc.expression.source
       else
         node
       end
-    CodeRay.scan(output, :ruby).term
+
+    code = CodeRay.scan(output, :ruby)
```

Comment:
I never need to focus on performance, but I think we can skip processing the scanning part if we're not going to colorize and use `output` directly.
```suggestion
  return output unless colorize
```